test/project_types/extension/commands/register_test.rb

```diff
@@ -98,6 +98,8 @@ def test_creates_the_extension_if_user_confirms
         ])
       end
 
+      private
```

Comment:
By making this to private, it indicates to that `run_register_command` is a helper and not a test.